src/main/java/pom/NaptolShippingPage.java

```diff
@@ -0,0 +1,103 @@
+package pom;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
+
+public class NaptolShippingPage {
+	@FindBy(xpath="//select[@name='fktitle_id']")private WebElement title;
+	@FindBy(xpath="//input[@name='firstName']")private WebElement firstName;
+	@FindBy(xpath="//input[@name='lastName']")private WebElement lastName;
+	@FindBy(xpath="//textarea[@name='address']")private WebElement completeAddress;
+	@FindBy(xpath="//input[@name='landmark']")private WebElement landmark;
+	@FindBy(xpath="//input[@name='pincode']")private WebElement pincode;
+	@FindBy(xpath="//select[@name='state']")private WebElement selectState;
+	@FindBy(xpath="//select[@name='city']")private WebElement selectCity;
+	@FindBy(xpath="//input[@name='mobile']")private WebElement mobileNumber;
+	@FindBy(xpath="//A[@id='addEditButton']")private WebElement saveAndProceed;
+	
+	public NaptolShippingPage(WebDriver driver) {
+		PageFactory.initElements(driver, this);
+	}
+	
+	  public void ClickOnTitle() {
+		  title.click();
+		  }
+
+      public void SelectTitle(WebDriver driver, String value) {
```

Comment:
No use of driver argument value in method

Reply:
Done.